.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -38,6 +38,16 @@ on:
       ref:
         type: string
         required: True
+      devops_ref:
+        type: string
+        required: False
+        description: |
+          By default we checkout the devops directory from "inputs.ref" branch.
```

Comment:
do we ever want this to be different? shouldnt we always use the current version from the `sycl` branch?